MarbleRally/2016/overallStandings.csv

```diff
@@ -0,0 +1,22 @@
+Racer,Diff,Rank,Pts,Rewards
```

Comment:
Looking at MarbulaOne/S01/overallStandings.csv might provide a better guide on how to do an overall standings CSV.

The headers for that league are:
Team,Gold,Silver,Bronze,FastestLaps,Points

In the case of the Marble Rally, I'd suggest:
Racer,Winner,Runner-up,3rd,Points

Data would be:
Racer Name,Number of wins,number of runner ups,number of third place finishes,total points

You can get most of this information from the wiki, but you'll need to do a little bit of formatting: 
https://jellesmarbleruns.fandom.com/wiki/Sand_Marble_Rally_2016#Standings

I'm OK if you choose to do the racer name instead of the Hashtag if it's easier to do.